autosubmit/autosubmit.py

```diff
@@ -5714,9 +5718,9 @@ def _change_conf(testid, hpc, start_date, member, chunks, branch, random_select=
 
         open(as_conf.experiment_file, 'wb').write(content)
 
-
+    # TODO: To be moved to utils
     @staticmethod
-    def load_job_list(expid, as_conf, notransitive=False, monitor=False, new = True): # To be moved to utils
+    def load_job_list(expid, as_conf, notransitive=False, monitor=False, new=True) -> JobList:
```

Comment:
Added return type, removed space to match the other args and fix linter/IDE warning, moved comment to the top with a TODO (I didn't write the part about moving to utils, at least not now)